tables/uniSwaps.go

```diff
@@ -0,0 +1,138 @@
+package tables
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"strings"
+)
+
+type UniSwapsTable struct{}
+
+func (tbl UniSwapsTable) GetID(r AggEvent) string {
+	return r.ID
+}
+
+func (tbl UniSwapsTable) GetTime(r AggEvent) int {
+	return r.Time
+}
+
+
+
+type UniSwapSubGraph struct {
+	ID              string `json:"id"`
+	Transaction	 struct {
+		ID string `json:"id"`
+		BlockNumber string `json:"blockNumber"`
+	}`json:"transaction"`
+
+	// EventIndex      int    `json:"eventIndex"`
```

Comment:
what is EventIndex /  should we be saving it?

Reply:
It's something that only existed on the crocswap subgraph - has no meaning in uniswap